tripy/tripy/flat_ir/ops/reshape.py

```diff
@@ -15,21 +15,52 @@
 # limitations under the License.
 #
 
+import array
 from dataclasses import dataclass
+from typing import Optional, Sequence
 
 from mlir_tensorrt.compiler.dialects import stablehlo
+from mlir_tensorrt.compiler.dialects._ods_common import get_op_result_or_value
 from mlir_tensorrt.compiler import ir
 
 from tripy.flat_ir.ops.base import BaseFlatIROp
 from tripy.backend.mlir.utils import is_any_dim_dynamic
 import tripy.utils.utils as utils
 
 
+def get_constant_value(arg) -> Optional[ir.DenseElementsAttr]:
```

Comment:
This seems more suited to live in `utils.py`

Reply:
Moved to utils.